pontoon/base/utils.py

```diff
@@ -667,18 +673,46 @@ def build_translation_memory_file(creation_date, locale_code, entries):
 
 
 def glob_to_regex(glob):
-    """This util uses python's fnmatch to convert a glob to a regex in a way
-    that can then be used with django's `__regex` queryset selector.
-
-    It prefixes the regex with `^`, and replaces the more complex match ending
-    provided by fnmatch, with the simpler `$`
-
     """
-    regex = "^%s" % fnmatch.translate(glob)
-    return (
-        "%s$" % regex[:-7]
-        if regex[-7:] == "\\Z(?ms)"
-        else regex)
+    This util uses PatternParser from compare_locales to convert a glob to a regex in a way
+    that can then be used with django's `__regex` queryset selector.
+    Supported wildcard operators:
+    - *
+    - **
+    """
+    pattern = PatternParser().parse(glob)
+    regex = r""
+
+    for part in pattern:
+        if isinstance(part, Starstar):
+            regex += r"(.*)"
+        elif isinstance(part, Star):
+            regex += r"([^/]*)"
+        elif isinstance(part, Variable):
+            # Variables are unsupportedignored because
```

Comment:
I'd raise an exception here instead of skipping.

Reply:
I was wondering about it too. Do you think that the error should be displayed in the Pontoon admin panel?